qemu/tests/virtio_fs_multi_vms.py

```diff
@@ -234,3 +231,12 @@ def get_viofs_exe(session):
                                       'virtiofs', session=session)
         if len(md5_set) != 1:
             test.fail('The md5 values are different among VMs.')
+
+    # for windows guest, disable/uninstall driver to get memory leak based on
+    # driver verifier is enabled
+    if os_type == "windows":
+        for vm, info in mapping.items():
+            session = info['session']
```

Comment:
Looks like there is no such place to call this variant `session` in the following codes.
So I think `session = info['session']` could be removed.

Reply:
You were right, I removed this line, thanks.